homeassistant/components/tankerkoenig/base_sensor.py

```diff
@@ -0,0 +1,82 @@
+"""Base classe for tankerkoenig sensor integration."""
+import logging
+
+from homeassistant.helpers.entity import Entity
+from homeassistant.const import (
+    ATTR_ATTRIBUTION,
+    ATTR_LATITUDE,
+    ATTR_LONGITUDE,
+    STATE_CLOSED,
+    STATE_OPEN,
+)
+from .const import NAME
+
+_LOGGER = logging.getLogger(__name__)
+
+ATTR_ADDRESS = "address"
+ATTR_BRAND = "brand"
+ATTR_FUEL = "fuel_type"
+ATTR_IS_OPEN = "state"
```

Comment:
I am using the "STATE_CLOSED" and "STATE_OPEN" constants for this attribute, so that the attribute is "state: open" or "state: closed".
If I change this to "is_open", it sounds strange: "is_open: open" or "is_open: closed". Or do you mean I should have just a boolean? "is_open: True"?

Reply:
I renamed the attribute to "is_open" as requested